### PR TITLE
Review exercise 4

### PR DESCRIPTION
**Pull request by HumphreyYang**

### CODE REVIEW

File: day-04/exercise_set_4.ipynb [hunk 1]
```diff
@@ -426,31 +363,19 @@
...
-     },
-     "execution_count": 16,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "A = np.array([[1, 4], [5, 7]])\n",
     "A"
```
Comment (HengchengZhang) [inline day-04/exercise_set_4.ipynb:397]:
```suggestion
    "print(A)"
```

**Review by HengchengZhang — COMMENTED** [review]
Hi @HumphreyYang it looks all good to me.

File: day-04/exercise_set_4.ipynb (hunk 1 above)
Comment (HumphreyYang) [reply day-04/exercise_set_4.ipynb:397]: I guess this is OK, as we have mentioned this functionality in exercise 1 : )

Comment (HengchengZhang) [reply day-04/exercise_set_4.ipynb:397]: It just looks weird without outputs, but I agree that it should be fine.

**Review by Smit-create — APPROVED** [review]
Thanks @HumphreyYang

**Pull request merged**